integration-tests/test/shared/docker-compose.ts

```diff
@@ -18,6 +18,7 @@ const DEFAULT_SERVICES: ServiceNames[] = [
 ]
 
 export class DockerComposeNetwork {
+  /*eslint no-empty-function: "off"*/
```

Comment:
Doesn't this turn off the feature for the entire file as opposed to ignore it for a single line? Maybe [`eslint-disable-next-line`](https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments) is what we want?